### ISSUE TITLE
Widget Inspector Problem

### ISSUE THREAD

**Issue by xaakib** [flutter/devtools#6892]
<-- Please describe your problem here. Be sure to include repro steps. -->
When i selected inpector it is work on device but not movment in code, i can't see the error code what i select in device. please tell me how can i solved that?
![image](https://github.com/flutter/devtools/assets/64243974/bb22cf52-42fd-44b0-80b0-fbea81b6f3aa)

___
**DevTools version**: 2.28.2
**IDE**: VSCode
**Connected Device**:
CPU / OS: x64 (64 bit) android
Connected app type: Flutter native (debug build)
Dart Version: 3.2.0
Flutter Version: 3.16.0 / stable
Framework / Engine: db7ef5bf9f / 74d16627b9

**Comment by xaakib** [comment]
Brother same to you. How can i solved this problem?

**Comment by andrerpena** [comment]
Hey @jacob314 thanks a lot for looking at it and replying.

I'm already always using debug

```
    {
      "name": "Flutter Debug",
      "type": "dart",
      "request": "launch",
      "program": "lib/main_dev.dart",
      "flutterMode": "debug"
    },
```

And I'm not sure there is a debug button, it's only a run button

![image](https://github.com/flutter/devtools/assets/1581429/7c399f49-59e7-4ff5-8f6d-cd32d3294c7b)

Thanks again!

**Comment by DanTup** [comment]
@andrerpena is the behaviour you're seeing what I described at https://github.com/flutter/devtools/issues/6876#issuecomment-1848421137 (and the comment below with a video)? That's the behaviour I've always seen and believed to be correct.

If you mean something else, can you give exact steps for what you're doing/seeing? Thanks!

**Comment by xaakib** [comment]
[DanTup](https://github.com/DanTup) widget inpector working on vscode run debugging but not working run without debugging.  before it worked. but after update flutter then it not work.

**Comment by DanTup** [comment]
@xaakib 

> [DanTup](https://github.com/DanTup) widget inpector working on vscode run debugging but not working run without debugging. before it worked. but after update flutter then it not work.

Yep, that is the issue noted above at https://github.com/flutter/devtools/issues/6892#:~:text=The%20other%20issue%20with%20navigation%20not%20working%20is%20fixed%20in%20the%20pre%2Drelease%20Dart%20extension%20and%20should%20be%20patched%20into%20the%20stable%20extension%20next%20week.%20If%20you%20want%20to%20test%20it%20out%20to%20ensure%20it%20solves%20your%20issue%2C%20you%20can%20switch%20to%20it%20in%20VS%20Code%20here%3A

That issue should be resolved in the pre-release Dart-Code extension and will be fixed in stable next week. Please test with the pre-release extension if you want to verify the issue is resolved.

**Comment by xaakib** [comment]
yes. it worked , thank you very much.

**Comment by DanTup** [comment]
This issue is resolved in v3.78.2 of the Dart extension. Please update and click the "Reload" button if prompted.

![image](https://github.com/Dart-Code/Dart-Code/assets/1078012/eab7282f-0bcc-435b-8e27-b2a321a2d221)

![image](https://github.com/Dart-Code/Dart-Code/assets/1078012/8149c83d-08fd-4644-b0c9-f4afc7c328cb)
